docs/targets/sendgrid.md

```diff
@@ -1,159 +1,56 @@
+In the Target creation form, provide a name for the event Target, and add the following information:
 
-* By replacing the section starting at line [#18](https://github.com/triggermesh/aws-event-sources/blob/93df64d5b298cc73acfb93f835646cc1d06429c4/config/samples/awskinesis-sinkbinding.yaml#L18) with:
+* **Default sender name**: Assign a default sender name. (optional)
+* **Default sender email**: Assign a default sender email. (optional)
+* **Default recipient name**:  Assign a default recipient's  name. (optional)
+* **Default recipient email**: Assign a default recipient's email. (optional)
+* **API Secret**: Reference to a [TriggerMesh secret][tm-secret] containing an [API token][api] for authenticating requests
 
-```
-  sink:
-    ref:
-      apiVersion: targets.triggermesh.io/v1alpha1
-      kind: ConfluentTarget
-      name: triggermesh-confluent
-```
+![Sendgridtarget form](../images/sendgrid-target/create-bridge-2.png)
 
-* You can now deploy this .yaml file. This will complete the setup of the sinkbinding!
+After clicking the `Save` button, the console will self-navigate to the Bridge editor. Proceed by adding the remaining components to the Bridge.
 
-*****Please note that to deploy the [awskinesis-sinkbinding.yaml](https://github.com/triggermesh/aws-event-sources/blob/master/config/samples/awskinesis-sinkbinding.yaml) example there are other resources that need to be deployed. Please see the [DEVELOPMENT.md](https://github.com/triggermesh/aws-event-sources/blob/master/DEVELOPMENT.md) doc hosted there for full details***** 
+After submitting the bridge, and allowing some configuration time, a green check mark on the main _Bridges_ page indicates that the bridge with a Sendgrid event Target was successfully created.
 
+![Bridge status](../images/bridge-status-green.png)
 
-### Talking to the Sendgrid Target
+For more information about using SendGrid, please refer to the [SendGrid documentation][docs].
 
-* Depending on how you want to use the target. You can set up defaults for all the avalible paraments and pass absolutely nothing to have a static message sent to a defined user. Or you can pass in key:value params to dynamically set all of the variables.
+### Example
 
-An example of a Cloudevent being passed via a Curl command:
+The SendGrid event Target can consume events of any type.
 
-```
-curl -v "10.152.183.229" \
-       -X POST \
-       -H "Ce-Id: 536808d3-88be-4077-9d7a-a3f162705f79" \
-       -H "Ce-Specversion: 1.0" \
-       -H "Ce-Type: dev.knative.samples.helloworld" \
-       -H "Ce-Source: dev.knative.samples/helloworldsource" \
-       -H "Content-Type: application/json" \
-       -d '{"FromEmail":"Jeff@triggermesh.com.","ToEmail":"jeffthenaef@gmail.com", \
-         "FromName":"SendgridTarget","ToName":"endUser","Message":"hello"}'
-```
+This Target accepts a [JSON][ce-jsonformat] payload with the following properties:
```

Comment:
This is in direct contradiction with

> The SendGrid event Target can consume events of any type.

above, so you can delete the sentence I quoted.

If we don't have a notion of "Type" in this target yet, it's better to just document the JSON format instead of saying the target accepts "any type".

Reply:
but.. the sendgrid target **does** accept and will consume events of any type. Maybe I am just unclear on something. ill ping you :)

Reply:
I think what Antoine is saying is that maybe the CE type can be anything but the JSON payload cannot be anything since you have specific keys.

Reply:
looks like this is now corrected. so make your PR something real and not a DRAFT and then merge :)

Reply:
I did speak with @antoineco on the matter and I have made updates after our conversation. If it is not resolved as-is, we are very close.